samples/add-features-with-contingent-values/README.metadata.json

```
@@ -1,42 +1,48 @@
 {
-  "category": "Edit and Manage Data",
-  "description": "Create and add features whose attribute values satisfy a predefined set of contingencies.",
-  "formal_name": "AddFeaturesWithContingentValues",
-  "ignore": false,
-  "images": [
-    "add-features-with-contingent-values.png"
-  ],
-  "keywords": [
-    "coded values",
-    "contingent values",
-    "feature table",
-    "geodatabase",
-    "ArcGISFeatureTable",
-    "CodedValue",
-    "CodedValueDomain",
-    "ContingencyConstraintViolation",
-    "ContingentCodedValue",
-    "ContingentRangeValue",
-    "ContingentValuesDefinition",
-    "ContingentValuesResult"
-  ],
-  "language": "kotlin",
-  "redirect_from": [
-    "/android/latest/sample-code/add-features-with-contingent-values.htm"
-  ],
-  "relevant_apis": [
-    "ArcGISFeatureTable",
-    "CodedValue",
-    "CodedValueDomain",
-    "ContingencyConstraintViolation",
-    "ContingentCodedValue",
-    "ContingentRangeValue",
-    "ContingentValuesDefinition",
-    "ContingentValuesResult"
-  ],
-  "snippets": [
-    "src/main/java/com/esri/arcgismaps/sample/addfeatureswithcontingentvalues/MainActivity.kt",
-    "src/main/java/com/esri/arcgismaps/sample/addfeatureswithcontingentvalues/DownloadActivity.kt"
-  ],
-  "title": "Add features with contingent values"
+    "category": "Edit and Manage Data",
+    "description": "Create and add features whose attribute values satisfy a predefined set of contingencies.",
+    "formal_name": "AddFeaturesWithContingentValues",
+    "ignore": false,
+    "images": [
+        "add-features-with-contingent-values.png"
+    ],
+    "keywords": [
+        "coded values",
+        "compose",
+        "contingent values",
+        "feature table",
+        "geodatabase",
+        "geoview",
+        "mapview",
+        "toolkit",
+        "ArcGISFeatureTable",
+        "CodedValue",
+        "CodedValueDomain",
+        "ContingencyConstraintViolation",
+        "ContingentCodedValue",
+        "ContingentRangeValue",
+        "ContingentValuesDefinition",
+        "ContingentValuesResult"
+    ],
+    "language": "kotlin",
+    "redirect_from": [
+        "/android/latest/sample-code/add-features-with-contingent-values.htm"
+    ],
```

Redirects are helpful to use when a sample name (underlying URL) has changed. In this case, the sample name is the same but migrated to compose. No need to use redirect here: 

```suggestion
    "redirect_from": "",
```